### PR TITLE
3.8 tooltips typos

### PR DESCRIPTION
**Pull request by adri9valle**
Hi team,

This PR solves several typos in the tooltips.

Solves => https://github.com/wazuh/wazuh-splunk/issues/406

Regards,

### CODE REVIEW

**Comment by manuasir** [comment]
This issue also resolves #398

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/general/agents-general.html [hunk 1]
```diff
@@ -83,7 +83,6 @@
           class="wz-tooltip">
           Start time: {{ agentInfo.syscheck.start || 'Unknown'}}
           <br> End time: {{ agentInfo.syscheck.end || 'Unknown'}}
-          <br> Duration time: {{ agentInfo.syscheck.duration +' minutes' || 'Unknown'}}
```
Comment (jesusgn90) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/general/agents-general.html]: Why you deleted this? Take a look at the Kibana app, you must calculate it and then show it

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/general/agentsGeneralCtrl.js [hunk 2]
```diff
@@ -184,6 +186,13 @@ define([
       this.filters = this.currentDataService.getSerializedFilters()
       this.state.reload()
     }
+
+    getDateDifferent(start, end){
```
Comment (jesusgn90) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/general/agentsGeneralCtrl.js]:
```js
validateRange(data) {
    const result = {
      duration: 'Unknown',
      inProgress: false,
      end: data.end || 'Unknown',
      start: data.start || 'Unknown'
    };

    if (data.end && data.start) {
      result.duration = (new Date(data.end) - new Date(data.start)) / 1000 / 60;
      result.duration = Math.round(result.duration * 100) / 100;
      if (result.duration <= 0) {
        result.inProgress = true;
      }
    }

    return result;
  }
```

Comment (jesusgn90) [reply SplunkAppForWazuh/appserver/static/js/controllers/agents/general/agentsGeneralCtrl.js]: Also place this function in a service and don't repeat the same code twice

**Pull request merged**